tests/DemoCallExitTest.php

```diff
@@ -61,8 +61,8 @@ public function casesDemoFilesdataProvider()
         $files = [];
         $base_path = dirname(__DIR__) . DIRECTORY_SEPARATOR . 'demos';
         foreach ($directories as $dir) {
-            $dir_path = $base_path . '/' . $dir;
-            $pref = '/' . $dir . '/';
+            $dir_path = $base_path . DIRECTORY_SEPARATOR . $dir;
```

Comment:
`'/'` will do, but otherwise ok. I think you should have also instant merge access for these fixes. @romaninsh

Reply:
$dir_path is used for scandir function, so probably better using the dir seperator.

Reply:
@ibelar no, all php functions should accept `/` for any platform as a path separator